java/pom.xml

```diff
@@ -15,7 +15,7 @@
         <module>openmldb-jdbc</module>
         <module>openmldb-native</module>
         <module>openmldb-common</module>
-        <module>openmldb-nearline-tablet</module>
+        <!-- <module>openmldb-nearline-tablet</module> -->
```

Comment:
so we won't publish nearline-tablet together with others modules?

Reply:
Yes, all the code of nearline-tablet will be removed. I comment out this to speed-up the compile process and we should remove all the code later.